Gemfile

```diff
@@ -6,6 +6,8 @@ source 'https://rubygems.org'
 # VMDB specific gems
 #
 
+gem 'ci_reporter_rspec'
```

Comment:
This looks like something you could add to your local `Gemfile.dev.rb`.

But, I'll let @chrisarcand and @Fryguy comment on whether we want to add this to the base repository's Gemfile.